### PR TITLE
Add 'exo-run help' command

### PR DESCRIPTION
**Pull request by hugobho**
@kevgo

### CODE REVIEW

File: features/help.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  So that I can do the things I intended without having to look this up somewhere else.
+
+
+  Scenario: the user enters 'exo-run help'
+    When running "exo-run help" in the terminal
+    Then I see:
+    """
+    Usage: exo-run
+
+    Runs an Exosphere application when executed in a directory containing an 'application.yml' file.
```
Comment (kevgo) [inline features/help.feature]: Runs an Exosphere application.

Must be executed in the application directory.

File: features/steps/steps.ls [hunk 2]
```diff
@@ -50,6 +50,20 @@ module.exports = ->
     @process.wait expected-text, done
 
 
+  @When /^running "([^"]*)" in the terminal$/, (command, done) ->
+    app-dir := path.join process.cwd!, 'tmp'
+    fs.empty-dir-sync app-dir
```
Comment (kevgo) [inline features/steps/steps.ls]: Why do you delete everything in the directory that you run this in? That shouldn't be necessary here.

File: features/steps/steps.ls [hunk 3]
```diff
@@ -50,6 +50,20 @@ module.exports = ->
...
+    args = command.split ' '
+    args[0] = path.join process.cwd!, 'bin', args[0]
+    if process.platform is 'win32'
```
Comment (kevgo) [inline features/steps/steps.ls]: This is unimportant enough to be merged onto one line:

```ls
if process.platform is 'win32' then args[0] += '.cmd'

**Comment by trushton** [comment]
@kevgo @hugobho This should be good to go. I just merged in the changes from all of the recent work with Docker.

**Pull request closed without merging**